### ISSUE TITLE
--hdf-archive option

### ISSUE THREAD

**Issue by tmolteno** [tart-telescope/tart2ms#44]
parser.add_argument('--archive-search', required=False, default=None, nargs="*",
                        help="Query hdf5 files from the TART online archive. (-10:1:0 last ten minutes interval 1 min) -100:10:0 last 100 minutes every 10 minutes. START:INTERVAL_MIN:END arbitrary timestamps")

**Comment by bennahugo** [comment]
I think it is actually better to specify UTC start and end times if possible and/or LST spans

**Comment by tmolteno** [comment]
The major use will be to get something like the last hour of data, and this will make the scripting much more intuitive. 

    signal:-120:1:-1 
    
We intend to have <START> and <END> be specified initially as unix timestamps if they are positive.  Adding ISOFORMAT dates with UTC specified can be addressed with a separate PR once this is done.
